### DISCUSSION TITLE
Many paper cuts that can probably be fixed easily...

### DISCUSSION THREAD

**Discussion by jessehouwing** [community/community#13456] [Mobile] [answered]
I've found myself using the app more and more. It's a great addition to email and the notification bell on GitHub itself.

It lead me to a few wishes, missing features and bugs:
* On a release with many artifacts, only the top 25 is so are shown. It looks like the rest API may be returning paged results, but the app won't fetch the next set when scrolling. https://github.com/jessehouwing/azure-pipelines-tasks-zips/releases/tag/m202
* File names in releases are truncated without an option to long-press for details or slide from left to right. See same release as previous point.
* LFS files can't be downloaded from the app, instead the lfs pointer file contents are shown.
* I sometimes wish there was an 'open in browser' button. I now use the ... -> share to copy the link, open the browser and paste the link again. This is mostly when I need access to things not in the mobile app, like settings.
* When a pull request contains a 'deployed to environment' link, it's not working in the app. See for example: https://github.com/github/docs/pull/15354
* Sometimes I need to make a change that's too hard to do on my phone. In that case it would be nice if you could let me 'notify on desktop' the next time I open GitHub in the browser. Sort of a do later option. That would allow me to dismiss the notification in my phone, but not forget about it.
* I sometimes wish I could edit plaintext and markdown files in the app. Another reason I switch to browser view regularly.
* Pasting pictures from the android clipboard isn't supported. I'd love that feature!
* I'd love an actions pane in which I can trigger and approve actions and get pending approval notifications in the app.

**Comment by aconsuegra** [comment, marked as answer]
Thanks a lot for the feedback @jessehouwing! This is a really awesome list of small improvements that would make the app much better. I have created a bunch of internal issues to track your requests, and I hope we can tackle them soon.

**Comment by unique-EJ** [comment]
> It lead me to a few wishes, missing features and bugs:
> 
> ...
> * I sometimes wish there was an 'open in browser' button. I now use the ... -> share to copy the link, open the browser and paste the link again. This is mostly when I need access to things not in the mobile app, like settings.
> * Sometimes I need to make a change that's too hard to do on my phone. In that case it would be nice if you could let me 'notify on desktop' the next time I open GitHub in the browser. Sort of a do later option. That would allow me to dismiss the notification in my phone, but not forget about it.
> * I sometimes wish I could edit plaintext and markdown files in the app. Another reason I switch to browser view regularly.

Yes - "in browser", "edit plain-text and markdown".
